app/controllers/MissingInfoController.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import actions.Actions
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
+import util.Logging
+
+import javax.inject.{Inject, Singleton}
+
+@Singleton
+class MissingInfoController @Inject() (
+    as:              Actions,
+    mcc:             MessagesControllerComponents,
+    missingInfoPage: views.html.MissingInformation
```

Comment:
Should this go into views and then use `views.missingInfoPage`. I'm not overly fussed if it doesn't but I think that's the pattern we have atm

Reply:
By views I mean the Views class we inject

Reply:
👍 I'll do that now

Reply:
just changed it in [this commit ](https://github.com/hmrc/essttp-frontend/pull/143/commits/85d367c7ae53ca5b0985bd793ceaf18bb235bad9)